ocaml/typing/typemodifier.mli

```diff
@@ -0,0 +1,14 @@
+type modifiers =
```

Comment:
The absence of a modifier is expressed via `None` rather than `Some max`. While this is not strictly necessary for the only user of this (`jkind.ml`), it makes the interface a little more general.